src/extension.ts

```diff
@@ -155,6 +155,7 @@ async function activate(context: vscode.ExtensionContext) {
 
   state.setExtensionContext(context);
   state.initDepsEdnJackInExecutable();
+  void depsClj.downloadDepsClj(context.extensionPath);
```

Comment:
Should this always be downloaded, even if clojure is set as the default deps edn jack-in executable by `initDepsEdnJackInExecutable`?

Reply:
There are still jack-in scenarios where deps.clj is used, regardless of that setting. The various Getting Started and Quick Start REPLs. (And I think we might still be using deps.clj on Windows.)

Reply:
Oh I see. Thanks.